### ISSUE TITLE
🐛  Frame Processor Error: runAtTargetFps is not a function (it is undefined), js engine: VisionCamera

### ISSUE THREAD

**Issue by VNDRN** [cuvent/react-native-vision-camera#2635]
### What's happening?

Using runAtTargetFps results in an error being thrown in the frameProcessor: 

> Frame Processor Error: runAtTargetFps is not a function (it is undefined), js engine: VisionCamera

#### App.tsx

```tsx
import { useEffect, useMemo } from "react";
import { StyleSheet, Text, View, useWindowDimensions } from "react-native";
import { useTensorflowModel } from "react-native-fast-tflite";
import {
  Camera,
  runAtTargetFps,
  useCameraDevice,
  useCameraPermission,
  useFrameProcessor,
} from "react-native-vision-camera";
import { useResizePlugin } from "vision-camera-resize-plugin";
import { useSharedValue } from "react-native-worklets-core";

export default function App() {
  const { hasPermission, requestPermission } = useCameraPermission();
  const device = useCameraDevice("back");
  const { resize } = useResizePlugin();

  const classificationMetaModel = useTensorflowModel(require("./assets/large075224classificationmetadata.tflite"));

  const result = useSharedValue("none");
  const { width, height } = useWindowDimensions();
  const cameraSize = Math.min(width, height);

  const { model } = classificationMetaModel;

  const frameProcessor = useFrameProcessor(
    (frame) => {
      "worklet";
      if (!model) return;
      runAtTargetFps(5, () => {
        "worklet";
        console.log("Let's classify the frame");
        const resizedFrame = resize(frame, {
          scale: {
            width: 224,
            height: 224,
          },
          pixelFormat: "rgb",
          dataType: "float32",
        });

        const outputs = model.runSync([resizedFrame]);
      });
    },
    [model]
  );

  //FIXME: https://github.com/mrousavy/react-native-vision-camera/issues/2293
  const format = useMemo(() => {
    const desiredWidth = 1920;
    const desiredHeight = 1080;
    if (device) {
      for (let index = 0; index < device.formats.length; index++) {
        const format = device.formats[index];
        if (format) {
          if (format.videoWidth == desiredWidth && format.videoHeight == desiredHeight) {
            return format;
          }
        }
      }
    }
    return undefined;
  }, [device?.formats]);

  useEffect(() => {
    const handlePermissions = async () => {
      await requestPermission();
    };
    if (!hasPermission) {
      handlePermissions();
    }
  }, [hasPermission, requestPermission]);

  return (
    <View style={styles.container}>
      {hasPermission ? (
        <Camera
          style={{ width: cameraSize, height: cameraSize }}
          device={device}
          isActive={true}
          enableFpsGraph
          frameProcessor={frameProcessor}
          format={format}
        />
      ) : (
        <Text>Permission not granted</Text>
      )}
      <Text style={{ padding: 8, fontSize: 32, color: "red", backgroundColor: "black" }}>{result.value}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
  },
});
```

#### package.json

```json
{
  "name": "expovision",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo run:android",
    "ios": "expo run:ios",
    "web": "expo start --web"
  },
  "dependencies": {
    "expo": "50.0.8",
    "expo-dev-client": "3.3.9",
    "expo-status-bar": "1.11.1",
    "react": "18.2.0",
    "react-native": "0.73.4",
    "react-native-fast-tflite": "1.2.0",
    "react-native-reanimated": "3.6.2",
    "react-native-vision-camera": "3.9.1",
    "react-native-worklets-core": "0.3.0",
    "vision-camera-resize-plugin": "2.0.1",
    "@types/react": "~18.2.45"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0",
    "typescript": "^5.3.0"
  },
  "private": true
}
```

### Relevant log output

```shell
Frame Processor Error: runAtTargetFps is not a function (it is undefined), js engine: VisionCamera
```

### Camera Device

```json
{
  "formats": [],
  "sensorOrientation": "landscape-left",
  "hardwareLevel": "full",
  "maxZoom": 10,
  "minZoom": 1,
  "maxExposure": 12,
  "supportsLowLightBoost": false,
  "neutralZoom": 1,
  "physicalDevices": [
    "wide-angle-camera"
  ],
  "supportsFocus": true,
  "supportsRawCapture": true,
  "isMultiCam": false,
  "minFocusDistance": 8,
  "minExposure": -12,
  "name": "BACK (0)",
  "hasFlash": true,
  "hasTorch": true,
  "position": "back",
  "id": "0"
}
```

### Device

onePlus Nord

### VisionCamera Version

3.9.1

### Can you reproduce this issue in the VisionCamera Example app?

No, I cannot reproduce the issue in the Example app

### Additional information

- [X] I am using Expo
- [X] I have enabled Frame Processors (react-native-worklets-core)
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [x] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by mrousavy** [comment]
Did you import it?

**Comment by VNDRN** [comment]
Yes! I've updated the reproducable code with my entire App.tsx file. I've also added my package.json

**Comment by mrousavy** [comment]
Hey - I think this issue has been fixed in [VisionCamera 4.0.0](https://github.com/mrousavy/react-native-vision-camera/releases/tag/v4.0.0). 🥳 

Please try V4 and let me know if you still experience this issue;
- if not, please consider [💖 **sponsoring me on GitHub** 💖](https://github.com/sponsors/mrousavy/) to support the development of VisionCamera and thank me for my time spent on fixing bugs and building new features.
- if you still see this issue, please comment and we can re-open this. But please update your native logs with the native (Xcode/Android Studio) logs from running VisionCamera V4 so I can investigate this.
